README.md

```diff
@@ -70,19 +70,23 @@ Set up a new local CMS instance.
 
 3. Build & Start the Docker Containers:
 
-    ```sh
-    make start
-    ```
+    | For Testing | For Developing & Testing |
+    | - | - |
+    | `make start` | `docker-compose -f ./docker-compose.dev.yml up` |
```

Comment:
Some development still requires `docker-compose -f ./docker-compose.dev.yml`[^1] e.g. [Locally Develop CMS and Styles](https://github.com/TACC/Core-CMS/wiki/Locally-Develop-CMS-and-Styles).

[^1]: Eventually, I might be able to replace it with `make start-dev`, but not yet.